image/spec/v1.1.md

```diff
@@ -99,7 +99,7 @@ This specification uses the following terms:
         <code>my-app:3.1.4</code>, <code>my-app</code> is the <i>Repository</i>
         component of the name. A repository name is made up of slash-separated
         name components, optionally prefixed by a DNS hostname. The hostname
-        must follow comply with standard DNS rules, but may not contain
+        must comply with standard DNS rules, but may not contain
```

Comment:
My preference would go to "comply with" as well, but wanted to check with @aaronlehmann, the devil may be in the detail with specs